docs/contributing/pair-programming.md

```diff
@@ -1,40 +1,40 @@
 ---
-title: Open Community Pair Programming Sessions
+title: Sessões de programação em pares abertas à comunidade
 ---
 
-The best part of open source is the community, and every community is stronger when it works together. To help build the strongest community possible, Gatsby is offering free [one-hour pair programming sessions][cal] to anyone and everyone in the open source community.
+A melhor parte do _open source_ é a comunidade, e cada comunidade é mais forte quando trabalha em conjunto. Para ajudar a construir a comunidade mais forte possível, o Gatsby está oferecendo [sessões de programação em pares de uma hora][cal] para qualquer pessoa e todos na comunidade de código aberto.
```

Comment:
Acho que traduzir o "anyone and everyone" do jeito literal como você fez não faz tanto sentido.

Sugiro: "Está oferencendo, gratuitamente, sessões de programação em pares de uma hora para todos da comunidade de código aberto".

Reply:
Sugestão:  "com duração de uma hora" pode ajudar na legibilidade.

Reply:
Se você não é uma pessoa que tem o hábito de contribuir na comunidade de open source, você não vai poder participar mais então é isso? Não concordo. A minha interpretação é que todos, sem restrições, não somente a quem faz parte da comunidade open source, mas  extendendo a todos da comunidade.

Reply:
@abnersajr "anyone and everyone" é uma expressao idiomática cujo objetivo é "exagerar" o significado que normalmente seria comunicado se fosse seria utilizado qualquer um dos dois termos em separado.  Concordo com o @rafaeelaudibert  que a separação é desnecessária, já que incomum na lingua portuguesa. 
Confira: 
https://ell.stackexchange.com/questions/61717/who-can-clarify-the-the-usage-of-anyone-and-everyone
[Anglicismo(https://pt.wikipedia.org/wiki/Anglicismo#Palavras_inglesas_aportuguesadas_(de_uso_aceito_em_geral))